e2e/workspace/WORKSPACE

```diff
@@ -16,6 +16,11 @@ nodejs_register_toolchains(
     node_version = DEFAULT_NODE_VERSION,
 )
 
+# NB: you can skip these "bazel_features" lines, so long as you call `npm_translate_lock`.
```

Comment:
Do we want to keep it in `npm_translate_lock` long term? I think we'd want a `rules_js_deps()` instead...

Reply:
I think we do, yes. WORKSPACE users shouldn't have an extra thing leaking out, there should only need to be two calls, "fetch deps" and "initialize". Plus, WORKSPACE is the "legacy way" to be removed by Bazel 9 so we don't care that much about changing this setup.

Reply:
Can we not wrap this in a `rules_js_deps()` though?

Then leave the call in `npm_translate_lock` to prevent a breaking change for most people, but I'd say drop that in 2.0 if we add `rules_js_deps`...

Reply:
Well, I just would rather not add yet more surface area to the WORKSPACE setup, ever. Under bzlmod this works great, and if we add a `rules_js_deps()` call then it's just another thing to document, and would never (?) be used for another purpose.
@gregmagolan WDYT?

Reply:
Do we need to make our dependency on bazel_features public though? Can we wrap it in our own `_deps` method even if we call it here?

Reply:
Given the legacy WORKSPACE way is going to be removed from Bazel v9 in approx 1 year I'd just go with Alex's hack inside npm_translate_lock and not add a thing that is going to be obsolete in 1 year. 99.9% of users will have that. I'm suddenly really looking forward to Bazel 9.

Reply:
cool, thanks.